From 27c8aff59e5cdabab96e5f02f3b52f326703235d Mon Sep 17 00:00:00 2001
From: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
Date: Mon, 16 Jul 2018 13:24:13 -0500
Subject: [PATCH 1338/1619] ASoC: SOF: default should not be module

Not sure why this was added but this is not quite right

Signed-off-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
---
 sound/soc/sof/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sound/soc/sof/Kconfig b/sound/soc/sof/Kconfig
index ba54d82..322cf65 100644
--- a/sound/soc/sof/Kconfig
+++ b/sound/soc/sof/Kconfig
@@ -6,7 +6,6 @@ config SND_SOC_SOF_ACPI
 
 config SND_SOC_SOF
 	tristate "Sound Open Firmware Support"
-	default m
 	select SND_SOC_TOPOLOGY
 	select SND_SOC_COMPRESS
 	help
-- 
2.7.4

